Assets/HoloToolkit/Utilities/Scripts/FpsDisplay.cs

```diff
@@ -12,16 +12,26 @@ namespace HoloToolkit.Unity
     [RequireComponent(typeof(TextMesh))]
     public class FpsDisplay : MonoBehaviour
     {
+        /// <summary>
+        /// Reference to TextMesh component where the FPS should be displayed.
+        /// </summary>
         [Tooltip("Reference to TextMesh component where the FPS should be displayed.")]
         [SerializeField]
         private TextMesh textMesh;
 
+        /// <summary>
+        /// Reference to uGUI text component where the FPS should be displayed.
+        /// </summary>
         [Tooltip("Reference to uGUI text component where the FPS should be displayed.")]
         [SerializeField]
         private Text uGUIText;
 
+        /// <summary>
+        /// How many frames should we consider into our average calculation?
+        /// </summary>
         [Tooltip("How many frames should we consider into our average calculation?")]
         [SerializeField]
+        [Range(1, 300)]
```

Comment:
Why 300?

Reply:
it's 5 seconds at 60fps. no reason other than that.  this field feels like it needs a range value. do you have a suggested max number of frames?

Reply:
Nope just curious